client/friends/friends.html

```diff
@@ -0,0 +1,24 @@
+<ion-header-bar align-title="left" class="bar-positive" style="background-color: #669999;">
+   <h1 class="title" style="margin-left:1em;">Friends List</h1>
+</ion-header-bar>
+<ion-content>
+   <ul class="list no-border p-b">
+      <li class="list-item" ng-repeat="friends in friendsList" ng-click="friendClick($index)">
+         <a herf="" class="list-left"><span class="w-40 avatar"><img src="{{friends.picture.data.url}}" alt="..."> <i class="on b-white bottom"></i></span></a>
+         <button class="button btn-sx pull-right invite-btn" ng-click="friends.invite()" ng-class="{shown: $index == selected}">
+         Invite
+         </button>
+         <div class="list-body">
+            <div><a href="">{{friends.name}}</a></div>
+            <small class="text-muted text-ellipsis">Designer, Blogger</small>
+         </div>
+      </li>
+      <li class="list-item">
```

Comment:
no sir only one friend is hard coded in to this page .. just not to show blank page ... will remove it .. friends using this app is now showing on this page